### PR TITLE
Visitor can buy a subscription

### PR DESCRIPTION
**Pull request by Cma-B**
### User Story
```
As a visitor
In order to get access to the best articles
I would like to be able to buy a subscription
```
## Changes proposed in this pull request:
* adds buying subscription for user 
* adds more styling
## What I have learned working on this feature:
* learned about hooks
* changing class component to function
## Screenshots (Client)
Screen Shot 2020-08-21 at 11 31 19 AM
Screen Shot 2020-08-21 at 11 37 18 AM
Screen Shot 2020-08-21 at 11 39 03 AM

### CODE REVIEW

**Review by Odeane — APPROVED** [review]
Looks great..Nice styling also

File: cypress/integration/userCanBecomeSubscriber.feature.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    });
+
+    cy.visit("/");
+    cy.get("#header-login").click();
+    cy.get("#login-form").within(() => {
+      cy.get("#email").type("registered@mail.com");
+      cy.get("#password").type("password");
+      cy.get("#login-submit").click();
+    });
+
```
Comment (Alex-Bellamy) [inline cypress/integration/userCanBecomeSubscriber.feature.js:29]: indention here and above

**Review by Felix-Willborg — APPROVED** [review]
Looks good to me.

**Review by Alex-Bellamy — COMMENTED** [review]
still flagged with WIP 
formatting in userCanBecomeSubscriber spec file could be updated to make it consistent throughout the file

**Pull request merged**